.github/workflows/continuous-integration.yml

```diff
@@ -190,7 +190,7 @@ jobs:
           - "default"
           - "3@dev"
         mariadb-version:
-          - "10.9"
+          - "lts"
```

Comment:
Isn't lts a placeholder for the last supported lts? When MariaDB changes the actual version behind "lts" doesn't ORM support it automatically.

Reply:
> Isn't lts a placeholder for the last supported lts?

yes.

> When MariaDB changes the actual version behind "lts" doesn't ORM support it automatically.

You'd hope so, but that hope need to be enforced by the reality of CI.

Most important - 10.9 is out of [maintenance](https://mariadb.org/about/#maintenance-policy) and should no longer be used.

Reply:
I'm fine with switching to maintained MariaDB versions in CI, but I agree that it should remain in our control which releases we actually test against. I don't think we should use the `lts` tag.

Reply:
done - 11.4